sass/_components.scss

```diff
@@ -654,4 +654,532 @@
+//
+// markup:
+// <div class="product-hero product-hero--3x1 {$modifiers}">
+//      <div class="columns is-mobile">
+//          <div class="column">
+//              <h2><span>What the hell is real vision television?</span></h2>
+//          </div>
+//      </div>
+//      <div class="columns">
+//          <div class="column product-hero__cell-container">
+//              <div class="product-hero__cell">
+//                  <h3><span>Unbiased</span></h3>
+//                  <p><span>Financial Insight</span></p>
+//              </div>
+//              <div class="product-hero__cell">
+//                  <h3><span>Unfiltered</span></h3>
+//                  <p><span>Actionable Ideas</span></p>
+//              </div>
+//              <div class="product-hero__cell">
+//                  <h3><span>Unapologetic</span></h3>
+//                  <p><span>Smart Opinions</span></p>
+//              </div>
+//          </div>
+//          <div class="column has-text-centered hero-cta__container">
+//              <a href="#" class="button">Read More</a>
+//          </div>
+//      </div>
+// </div>
+// 
+// Styleguide 5.11
+.product-hero {
+    &.product-hero--3x1 {
+        background: url('#{$root}/images/components/product-hero-bg.png');
+
+        h2 {
+            margin-bottom: $baseline;
+        }
+
+        .product-hero__cell-container {
+            padding-bottom: $baseline / 2;
+
+            @include mobile {
+                padding-bottom: 0;
+            }
+            .product-hero__cell {
+                margin-bottom: $baseline;
+            }
+        }
+    }
+    position: relative;
+    background-size: cover;
+    background-repeat: no-repeat;
+    padding: $baseline ($baseline * 2);
+    text-transform: uppercase;
+
+    @include mobile {
+        padding: $baseline;
+    }
+
+    em {
+        font-style: normal;
+        color: $color-torch-red;
+    }
+
+    h2, h3 {
+        margin-bottom: $baseline * 2;
+        span {
+            margin: 0;
+            font-weight: 800;
+        }
+        @include mobile {
+            margin-bottom: $baseline;
+        }
+    }
+    h2 {
+        line-height: 49px;
+        span {
+            font-size: $size-2;
+            color: black;
+            background-color: white;
+            box-shadow: 10px 0 0 white, -10px 0 0 white;
+        }
+
+        @include mobile {
+            line-height: 24px;
+            span {
+                font-size: $size-4;
+            }
+        }
+    }
+    h3 {
+        line-height: 36px;
+        span {
+            font-size: $size-3;
+            color: white;
+            background-color: black;
+            box-shadow: 10px 0 0 black, -10px 0 0 black;
+        }
+
+        @include mobile {
+            line-height: 26px;
+            span {
+                font-size: $size-4;
+            }
+        }
+    }
+    p {
+        margin-top: 0;
+        margin-bottom: 0;
+        line-height: 28px;
```

Comment:
You have a few line heights in pixels - can you make them unitless or rems so that they scale based on the document font size?